scripts/rebuild-ui.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+cd ../jaeger-ui
+
+if git describe --exact-match --tags $(git rev-parse HEAD) >/dev/null 2>&1; then
+    
+    # Get the release version from the tag  
+    release_version = $(git describe --exact-match --tags $(git rev-parse HEAD)) 
+
+
+    # Check if the corresponding UI release has the assets.tar.gz file uploaded
+    release_url="https://github.com/jaegertracing/jaeger-ui/releases/download/${release_version}/assets.tar.gz"
+
+    temp_file=$(mktemp)
+
+    # Download the file to the temporary file and extract it into packages/build/
+    if curl --output /dev/null --silent --head --fail "$release_url"; then
+        curl --location --output "$temp_file" "$release_url"
+        tar -zxvf "$temp_file" -C packages/jaeger_ui/build/
+        rm -f "$temp_file"
+    
+    else
```

Comment:
rather than having two duplicate `else`, just call `exit 0` here. Something like:
```
tag=$(git describe ...)
if tag matches pattern; then
    temp_file=$(mktemp)
    trap "rm -f ${temp_file}" EXIT
    if curl ... ; then
        untar
        exit 0
    fi
fi

# do a regular full build
yarn install --frozen-lockfile && cd packages/jaeger-ui && yarn build
```